javascript/lesson4/exercise1/main.js

```diff
@@ -0,0 +1,17 @@
+function exercise1(){
+    var inventaire=["salade", "tomate", "oignons"];
+    console.log( inventaire.length);
+    console.log(inventaire[0]);
+    inventaire.push("sauce blanche");
+   console.log( inventaire.length);
+    inventaire.pop("sauce blanche");
+    afficher(inventaire);
+
+}
+
+exercise1();
+
+    function afficher(tab) {
+        for var i=0;i<inventaire.lenght; i++) 
+
```

Comment:
il faut des accolades apres la boucle for, pour indiquer ce qu'on va faire dans la boucle
et dans cette boucle, il faut afficher le contenu du tableau en utilisant l'itérateur "i" en tant qu'index